.github/workflows/esti.yaml

```diff
@@ -88,17 +88,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Setup NodeJS
-        uses: actions/setup-node@v3
-        with:
-          node-version: "16.17.1"
-
-      - name: Setup Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: 1.19.2
-        id: go
-
```

Comment:
Are these not needed because one of the docker images we use includes them?

Reply:
I don't know why they were here, they were not used the way I understand it.